source/MySettings.mc

```diff
@@ -585,6 +585,7 @@ class MySettings {
     return LangUtils.readKeyNumber(App.Properties.getValue("userLivetrack24Frequency"), 0);
   }
   function saveLivetrack24Frequency(_iValue as Number) as Void { 
+    setLivetrack24Frequency(_iValue);
```

Comment:
Can you describe the bug? The setLiveTrack24Frequency() function (and all the other set funcctions) is called when the user exits the settings screen via the MyApp.onSettingsChanged() function, thus updating the iLivetrack24FrequencySeconds variable.
So the out of sync of the iLivetrack24FrequencySeconds variable only occurs between the moment the user sets the new value, and the moment they exit the settings (which they need to do to use the app, so in practice it's never out of sync).

I'd like to remove this line (and the one for STL, and the one you also added for FS), unless you tell me I'm missing something!

Reply:
You are correct then, while I was developing I noticed that It didnt change instantly when exiting the picker, but I didnt try exiting the settings. I will remove the commit.